.github/workflows/comment-changes.yml

```diff
@@ -13,6 +13,7 @@ permissions:
 jobs:
   aeruginous:
     if: |
+      github.repository == 'AmmarAbouZor/tui-journal' &&
       !contains(github.event.head_commit.message, '[Aeruginous]') &&
       !contains(github.event.head_commit.message, 'Chore:') &&
       !contains(github.event.head_commit.message, 'Docs:')
```

Comment:
I tested the skip conditions; they only work if the keyword is contained in a commit's *summary* but not when it is its body.  As the latter case will be usual in this repository, these conditions are unlikely to be ever met.  However, they do no damage, so they can be left in for the case of a batch push of commits apart from PRs.

Reply:
For this case I'll use squash and merge for the pull requests that should be skipped